xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -3024,6 +3046,27 @@ struct AppModel: ModelProtocol {
         return Update(state: state, fx: fx)
     }
     
+    static func appendToEntry(
+        state: Self,
+        environment: AppEnvironment,
+        address: Slashlink,
+        append: String
+    ) -> Update<Self> {
+        let fx: Fx<AppAction> = Future.detached {
+            var entry = try await environment.data.readMemo(address: address)
+            entry.body.append(append)
+            try await environment.data.writeEntry(MemoEntry(address: address, contents: entry))
+            
+            return .succeedAppendToEntry(address: address)
+        }
+        .recover { error in
+            .failAppendToEntry(address: address, error.localizedDescription)
+        }
+        .eraseToAnyPublisher()
+        
+        return Update(state: state, fx: fx)
+    }
+    
```

Comment:
Generic machinery to append text to a note, I can see this being something we for more than links.